teacher.rb

```diff
@@ -1,7 +1,7 @@
 require './person'
 
 class Teacher < Person
-  attr_accessor :specialization
+  attr_reader :specialization
```

Comment:
- [ ] An `ArgumentError` occurs when attempting to create a teacher. Please try to fix the bug 👍 
![image](https://github.com/AFakhriddinov/School-library-OOP/assets/46839548/bf054f43-2cf2-44a1-b337-f7ad246b6130)

Reply:
Thanks for your feedback.